### ISSUE TITLE
Unwanted Space before punctuation when text is styled

### ISSUE THREAD

**Issue by justinsunho** [xuopled/gatsby-source-google-docs#25]
Because text is joined by a space when converting from google doc to json, a space is put in between punctuation and styled text when the doc is rendered.

e.g. **This is a test text**. ends up looking like this **This is a test text** . 

This can be avoided if the period is included in the styling: (**This is a test text.**), but I don't believe that most google doc users (marketers etc.) style their punctuation with the rest of the text.

I recommend a solution like this: 

```
const bulletContent = paragraph.elements
  .map(getText)
  .join(" ")
  .replace(" .", ".")
  .replace(" ,", ",")
```

and

```
content.push({
  [tag]: tagContent
  .map(el => el[tag])
  .join(" ")
  .replace(" .", ".")
  .replace(" ,", ","),
})
```

in `convert-google-document-to-json.js`

**Comment by xuopled** [comment]
Nice catch. Could you make a PR to fix it? Thank you very much

**Comment by xuopled** [comment]
Thank you. Fixed in `v1.11.0`
